src/view/range.js

```diff
@@ -82,6 +87,60 @@ export default class Range {
 	}
 
 	/**
+	 * Creates a new range with the same content, but containing all surrendering tags with no semantic meaning.
+	 *
+	 * For example:
+	 *
+	 * 		<p>Foo</p><p><b>{Bar}</b></p> -> <p>Foo</p>[<p><b>Bar</b></p>]
+	 * 		<p><b>foo</b>{bar}<span></span></p> -> <p><b>foo[</b>bar<span></span>]</p>
+	 *
+	 * Note that in the sample above all:
```

Comment:
remove "all", I understand that sentence as "the sample that is above all samples".